### PR TITLE
Update for digit support

### PR DESCRIPTION
**Pull request by mathcat4**
What I did is use another method to support digits.
Some minor changes:
- change TokenRE to RegEx because one is not a token
- changed class objects to title case (because I think I saw that somewhere in PEP 8
- changed MathOp and Add to a general Function, because in the tokenizer, you don't tokenize each function

Feel free to revoke any edit I made

### CODE REVIEW

**Comment by Tams-Tams** [comment]
solves issue #1

**Comment by TheSecondComing123** [comment]
A space is should not be ignored because without it, `+ 4 5` would be `+45`

**Comment by Tams-Tams** [comment]
Sorry for the bad naming, but I mean that a <Space> should not be added as a Token. Any other naming ideas?

**Comment by TheSecondComing123** [comment]
> Sorry for the bad naming, but I mean that a should not be added as a Token. Any other naming ideas?

I would name it `IgnoreToken`.

File: tokenizer.py [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
+    Function = "function"
 
 
-class TokenRe(enum.Enum):
-    NUMBER_RE = r"\d"
-    MATHOP_RE = r"[+\-*/]"
+class RegEx(enum.Enum):
+    Number = r"\d"
+    Function = r"\D"
+    Ignore = r" "
```
Comment (TheSecondComing123) [inline tokenizer.py:13]: I would prefer this be named "IgnoreToken"

**Pull request merged**